web/src/lib/i18n/en.json

```diff
@@ -64,8 +66,8 @@
     "job_settings": "Job Settings",
     "job_settings_description": "Manage job concurrency",
     "job_status": "Job Status",
-    "jobs_delayed": "{jobCount} delayed",
-    "jobs_failed": "{jobCount} failed",
+    "jobs_delayed": "{jobCount, plural, other {#}} delayed",
```

Comment:
Why use the plural form over `{jobCount, number}`? I've also seen this with a few other translations.

Reply:
When plural is not defined in the source language and it's used in target language Weblate reports item translation as forced ICU MessageFormat. Other languages can inflect "delayed" depending on job count.

Reply:
Understood. I think using `{# delayed}` in that case would communicate the intention a bit better, but that's just a minor thing

Reply:
You're right, `{# delayed}` is much better.